### PR TITLE
[14.0] Plug sale_order_import w/ EDI framework

### PR DESCRIPTION
**Pull request by simahawk**
Depends on / includes:

- [x] #470 
- [x] https://github.com/OCA/edi/pull/483

`edi_sale_order_import_ubl_endpoint` is meant to replace `sale_order_import_ubl_http` https://github.com/OCA/edi/pull/317
The module itself it's not required IMO but it will be an example on how to setup and endpoint for EDI exchanges.

### CODE REVIEW

**Comment by simahawk** [comment]
@sbidoul any idea of what's going wrong here on pre-commit run?
```
     ModuleNotFoundError: No module named 'setuptools'
```
https://github.com/OCA/edi/runs/4650728150?check_suite_focus=true#step:4:163

I've tried updating version as per https://github.com/OCA/oca-addons-repo-template/pull/100 but has no effect (as expected).

FTR I had to do the change manually because `copier update` does not pull latest changes here :man_shrugging:

**Comment by sbidoul** [comment]
@simahawk https://github.com/OCA/oca-addons-repo-template/pull/101

**Comment by simahawk** [comment]
trying w/out the pre-commit fix as suggested on OCA/oca-addons-repo-template#101

**Review by sebalix — APPROVED** [review]

File: edi_ubl_oca/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "EDI UBL",
+    "summary": """Define EDI backend type for UBL.""",
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/edi",
+    "author": "Camptcamp,Odoo Community Association (OCA)",
```
Comment (jbaudoux) [inline edi_ubl_oca/__manifest__.py]:
```suggestion
    "author": "Camptocamp,Odoo Community Association (OCA)",
```
:)

Comment (jbaudoux) [reply edi_ubl_oca/__manifest__.py]: @simahawk There are many typo like this, please review them all

**Review by jbaudoux — APPROVED** [review]
LGTM, no tests

File: edi_ubl_oca/__manifest__.py (hunk 1 above)
Comment (simahawk) [reply edi_ubl_oca/__manifest__.py]: good catch, I fixed in other modules in this repo https://github.com/OCA/edi/pull/514 I'll do it here too. Tnx!

**Pull request merged**